src/tmp.jl

```diff
@@ -375,8 +375,17 @@ function tmp_create_lowest_resolution_table(connection)
         current_group = ("", 0, 0)
         @timeit to "append $table_name rows" for row in DuckDB.query(
             connection,
-            "SELECT * FROM $union_table
-            ORDER BY asset, year, rep_period, time_block_start
+            "SELECT t_union.* FROM $union_table AS t_union
+            LEFT JOIN asset_both
+                ON asset_both.asset = t_union.asset
+                AND asset_both.milestone_year = t_union.year
+                AND asset_both.commission_year = t_union.year
+            WHERE asset_both.active = true
```

Comment:
same comment here as in https://github.com/TulipaEnergy/TulipaEnergyModel.jl/pull/962/files#r1882306145